documentation/installation.md

```diff
@@ -225,3 +224,102 @@ Debug
 To access the console, you can use `shell_plus`:
 
     make shell
+
+## OCIM Frontend
+
+The frontend of OCIM is a single-page app using the React and Redux. All the code can be found inside `frontend` directory.
+
+### Provisioning
+
+- Install the API client:
+
+```bash
+./scripts/build-api-client.sh
+```
+
+- Install requirements:
+
+```bash
+npm install
+```
+
+- Run frontend:
+
+```bash
+npm start
+```
+
+- Updating the API Client:
+
+```bash
+cd ./frontend # go to frontend directory
+npm run update-api-client
+```
+
+- Building the API Client:
+
+```bash
+cd ./frontend # go to frontend directory
+npm run build-api-client
+```
+
+- Deployment:
```

Comment:
This section doesn't relate to the new frontend. 
Why is `screen` mentioned here?

Reply:
Btw, the frontend deployment is done through CircleCI and happens automatically when we merge stuff into `master`, using CircleCI Contexts to change the build environment configuration.
The actual frontend bundle lives in S3 and is served through CloudFront.
See the `frontend-` tasks in circle.yml: https://github.com/open-craft/opencraft/blob/master/circle.yml#L7